HotTips/Engine/TipCalculator.cs

```diff
@@ -109,6 +111,12 @@ private TipInfo GetNextTipAlgorithm(string lastSeenGroupId)
                             // Assume tipInfo will not be null at this point.
                             TipInfo tipInfo = tipsAtPriBand[scanRow];
 
+                            // If the tip level is excluded, move on to the next group.
```

Comment:
No! If the tip level is excluded, you should find the next tip in that same group.